templates/new/config.yaml.txt

```diff
@@ -4,3 +4,12 @@ output:
 image: {project_name}
 company: yourcompany
 version: latest
+
+# Details of class inheriting from Wrapper
+wrapper-info:
```

Comment:
Could you please change this to `wrapperClass: {project_name}.wrapper.PipelineWrapper`?

Reply:
Done!